react-deploy-to-s3-action.yml

```diff
@@ -0,0 +1,26 @@
+# File: react-deploy-to-s3-action.yml
+# Sync a built React application to an AWS S3 repository.
+
+parameters:
+  - name: bucket
+    type: string
+
+steps:
+  - task: AWSShellScript@1
+    inputs:
+      awsCredentials: 'OUIT PROD'
```

Comment:
Should we pass this in as a parameter so that we can use it for both prod and dev?

Reply:
@nicholaswilson100 What did you think to this?

Reply:
Yeah you're right. - I didn't do it because the last thing I heard from Adam was that we weren't doing pipleines

Reply:
Yeah let me change this  - should do it properly I suppose even if it's not going to be used